dashboard/assets/js/cumulative-graph.js

```diff
@@ -224,7 +224,12 @@ export function plotCumulativeSpendings(project, showBurnDown, startDate, endDat
       yanchor: 'top'
     },
     shapes: shapes,
-    annotations: annotations
+    annotations: annotations,
+    margin: {
```

Comment:
`top`, `left` and `right` respectively. They default to 80px!

https://plot.ly/javascript/reference/#layout-margin